SonarQube.Bootstrapper/Resources.resx

```diff
@@ -138,6 +138,9 @@
   <data name="ERROR_ProcessingFailed" xml:space="preserve">
     <value>{0} failed. Exit code: {1}</value>
   </data>
+  <data name="ERROR_TempDirDoesNotExist" xml:space="preserve">
+    <value>Temporary analysis directory (usually .sonarqube) doesn't exist.</value>
+  </data>
```

Comment:
shouldn't the message give the tip that perhaps the begin step was not executed?